src/psyclone/tests/psyir/nodes/array_mixin_test.py

```diff
@@ -564,3 +564,127 @@ def test_get_outer_range_index_error():
     array = ArrayReference.create(symbol, [Literal("2", INTEGER_TYPE)])
     with pytest.raises(IndexError):
         _ = array.get_outer_range_index()
+
```

Comment:
I get:
```
psyir/nodes/array_mixin_test.py:569:0: R0915: Too many statements (59/50) (too-many-statements)
```
Maybe split this test into two, one for cases raising an exception, one for the rest?

Reply:
Done

Reply:
Thanks!